Name: Shorcut-keys-In-Browser
Version: 1.3.5-2
Author: Fritos1406
Description: Adds shorcut keys to the launcher. Hold onto Gesture+key to execute
New Card: Gesture+n
Add Bookmark: Gesture+m
Add To Launcher: Gesture+l
Bookmarks: Gesture+b
History: Gesture+h
Share Page: Gesture+s
Refresh Page: GEsture+r

--- .orig/usr/palm/applications/com.palm.app.browser/app/controllers/page-assistant.js
+++ /usr/palm/applications/com.palm.app.browser/app/controllers/page-assistant.js
@@ -787,8 +787,8 @@
 		this.appMenuModel = {
 			visible: true,
 			items: [
-				MenuData.ApplicationMenu.NewCard,
-				MenuData.ApplicationMenu.AddBookmark,
+				{label:$L('New Card'),     shortcut:'n', command:'new-page-cmd'},
+				{label:$L('Add Bookmark'),     shortcut:'m', command:'add-bookmark-cmd'},
 				{
 					label: $L("Page"),
 					items: [
@@ -792,8 +792,10 @@
 				{
 					label: $L("Page"),
 					items: [
-						MenuData.ApplicationMenu.AddToLauncher,
-						MenuData.ApplicationMenu.SharePage]
+						{label:$L('Add To Launcher'),     shortcut:'l', command:'add-launch-icon-cmd'},
+						{label:$L('Share Page'),     shortcut:'s', command:'share-page-cmd'},
+                       {label:$L('Refresh Page'),     shortcut:'r', command:'refresh'},
+                           ]
 				},
 				MenuData.ApplicationMenu.ShowBookmarks,
 				MenuData.ApplicationMenu.ShowHistory]
@@ -795,8 +796,9 @@
 						MenuData.ApplicationMenu.AddToLauncher,
 						MenuData.ApplicationMenu.SharePage]
 				},
-				MenuData.ApplicationMenu.ShowBookmarks,
-				MenuData.ApplicationMenu.ShowHistory]
+				{label:$L('Bookmarks'),     shortcut:'b', command:'show-bookmarks-cmd'},
+				{label:$L('History'),     shortcut:'h', command:'show-history-cmd'},
+                           ]
 		};
 
 		this.controller.setupWidget(Mojo.Menu.appMenu, undefined, this.appMenuModel);
